src/sidebar/components/SearchInput.js

```diff
@@ -87,7 +85,12 @@ export default function SearchInput({ alwaysExpanded, query, onSearch }) {
           />
         </div>
       )}
-      {isLoading && <Spinner />}
+
+      {isLoading && (
+        <div style="margin:0.5em 0">
```

Comment:
This might catch your eye. The styling for this component is in such need of refactor that I feel that it's less confusing just to be explicit about this div's styling needs versus creating a new classname.